README.md

```diff
@@ -1,149 +1,116 @@
-We've aimed to provide as much documentation as we can for the various components of Buf to give
-you a full understanding of Buf's surface area.
+# Buf
 
-Finally, [follow the project on GitHub](https://github.com/bufbuild/buf),
-and [contact us](https://docs.buf.build/contact/) if you'd like to get involved.
+[![License](https://img.shields.io/github/license/bufbuild/buf?color=blue)][badges.license]
+[![Release](https://img.shields.io/github/v/release/bufbuild/buf?include_prereleases)][badges.release]
+[![CI](https://github.com/bufbuild/buf/workflows/ci/badge.svg)][badges.ci]
+[![Docker](https://img.shields.io/docker/pulls/bufbuild/buf)][badges.docker]
+[![Homebrew](https://img.shields.io/badge/homebrew-v1.0.0--rc12-blue)][badges.homebrew]
+[![AUR](https://img.shields.io/aur/version/buf)][badges.aur]
+[![Slack](https://img.shields.io/badge/slack-buf-%23e01563)][badges.slack]
+[![Twitter](https://img.shields.io/twitter/follow/bufbuild?style=social)][badges.twitter]
+
+The [`buf`][buf] CLI is a tool for working with [Protocol Buffers][protobuf] APIs, offering a range of features not found in the standard `protoc` compiler, including:
+
+- The ability to manage Protobuf assets, including [plugins] and [templates], on the [Buf Schema Registry][bsr] (BSR).
+- A [linter][lint_usage] that enforces good API design choices and structure.
+- A [breaking change detector][breaking_usage] that enforces compatibility at the source code or wire level.
+- A [generator][generate_usage] that invokes your protoc plugins based on a configurable [template][templates].
+  A [protoc replacement][protoc] that uses Buf's [high-performance Protobuf compiler][compiler].
+- A configurable file [builder][build_usage] that produces [Images], our extension of Protobuf's native [FileDescriptorSets][filedescriptorset].
+
+See more [features] below.
+
+## Installation
+
+### Homebrew
+
+You can install `buf` using [Homebrew][brew] (macOS or Linux):
+
+```sh
+brew install bufbuild/buf/buf
+```
+
+This installs:
+
+* The `buf`, [`protoc-gen-buf-breaking`][breaking], and [`protoc-gen-buf-lint`][lint] binaries
+* Shell completion scripts for [Bash], [Fish], [Powershell], and [zsh]
+
+### Other methods
+
+For other installation methods, see our [official documentation][install], which covers:
+
+* Installing `buf` on [Windows]
+* Using `buf` as a [Docker image][docker]
+* Installing as a [binary], from a [tarball], and from [source] through [GitHub Releases][releases]
+* [Verifying] releases using a [minisign] public key
+
+## Usage
+
+Buf's help interface provides comprehensive docs:
+
+```sh
+buf --help
+```
+
+For more comprehensive info, consult Buf's [documentation][docs] for usage information, especially these guides:
+
+* [`buf breaking`][breaking_usage]
+* [`buf build`][build_usage]
+* [`buf generate`][generate_usage]
+* [`buf lint`][lint_usage]
+* [`buf registry`][bsr_usage] (for using the [BSR])
```

Comment:
This command doesn't exist

Reply:
It does: https://github.com/bufbuild/buf/blob/main/private/buf/cmd/buf/buf.go#L126-L132

Reply:
The non-beta ones are `login` and `logout` -- I'm think I would prefer not to point to `beta` commands here (but just my opinion).